Makefile

```diff
@@ -126,17 +131,20 @@ cleanup_remote:
 
 # end of image build targets which have been moved to docker/Makefile
 
-tests: build
-	$(MAKE) get_test_data
-	$(MAKE) run_tests_sequential
-
 test: tests
 
-tests_mpi: build
+tests:
+	PYTEST_CMD="$(PYTEST_MAIN)" $(MAKE) test_base
+
+savepoint_tests:
 	$(MAKE) get_test_data
-	$(MAKE) run_tests_parallel
+	VOLUMES='-v $(TEST_DATA_HOST):$(TEST_DATA_RUN_LOC)' \
+	PYTEST_CMD="$(PYTEST_SEQUENTIAL)" $(MAKE) test_base
 
-test_mpi: tests_mpi
+savepoint_tests_mpi:
+	$(MAKE) get_test_data
+	VOLUMES='-v $(TEST_DATA_HOST):$(TEST_DATA_RUN_LOC)' \
```

Comment:
shouldn't these VOLUMES be a += rather than overwriting? What about the VOLUME you set in the jenkins action or the DEV above?

Reply:
Yes, good catch! It looks like this was a pre-existing bug, which will now affect us because I'm using VOLUMES= from the command-line. Will fix.

Reply:
I don't think it is a bug in master as VOLUMES isn't used outside of the Makefile and is set to different values for the different endpoints, but with them combined is great o have them be additive

Reply:
Yes the "bug" in master is that it's settable outside the Makefile (uses `?=` and not `=`) but if you set it then in theory it will be ignored/overridden. In practice we've never set it outside the Makefile so it's never affected us.

Reply:
The use case would have been running VOLUMEs=<> make test_base separately with some custom volumes, while the other endpoints hardcoded it and indeed couldn't add to it or override it. It was intended behavior.

Reply:
(that is no longer needed)